src/components/Game/Backspace/Backspace.scss

```diff
@@ -0,0 +1,41 @@
+:global {
+    @import "@fortawesome/fontawesome-free/scss/fontawesome";
+    @import "@fortawesome/fontawesome-free/scss/solid";
+}
+
+@import "../../../styles/abstracts/breakpoints";
+@import "../../../styles/abstracts/variables";
+
+.Backspace {
+    justify-self: self-end;
+}
+
+.Button {
```

Comment:
it's the only button in this component, which is in general all about this button. in a second it will be the only element in this component (I agree with what you wrote about the nesting). along with the fact we are using CSS Modules is there a need for this class name to be more specific?

Reply:
depends on your preferences :)
the name `abcdefgh` will work too but is it good?

Reply:
I'm merging it with `.Backspace` class anyway (due to refactoring in `Backspace.js`) so the problem is kind of resolving itself ;p